config/src/main/java/no/sysco/middleware/kafka/clients/interceptor/config/ConfigHarvesterInterceptor.java

```diff
@@ -0,0 +1,173 @@
+package no.sysco.middleware.kafka.clients.interceptor.config;

```

Comment:
Could we move this to `n.s.m.k.interceptor.config` to keep it align with other interceptors?

Reply:
fixed